src/index.css

```diff
@@ -96,6 +97,7 @@ input:focus {
 }
 
 .postColumn:hover {
+  color: white;
```

Comment:
that makes it buggy for light mode, use variables in both changes please

Reply:
![image](https://github.com/narayan954/dummygram/assets/77617189/7fb18481-d242-440d-8b14-d2082a7ede0e)![image](https://github.com/narayan954/dummygram/assets/77617189/ea4a49cc-d007-4d53-bcb6-4d655b47d11e)